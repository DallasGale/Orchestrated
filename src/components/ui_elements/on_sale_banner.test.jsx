import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import OnSaleBanner from './on_sale_banner'

describe('<OnSaleBanner />', () => {
  test('it renders', () => {
    const Component = render(<OnSaleBanner />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
