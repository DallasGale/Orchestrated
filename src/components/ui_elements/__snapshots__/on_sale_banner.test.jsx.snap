// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OnSaleBanner /> it renders 1`] = `
<div
  class="sc-ifAKCX gnKWoA"
>
  <div
    class="sc-EHOje eBeZJa"
  >
    ON SALE!
  </div>
</div>
`;
